packages/@coorpacademy-components/src/util/external-content.js

```diff
@@ -0,0 +1,12 @@
+const isExternalContent = type => {
+  switch (type) {
+    case 'scorm':
+    case 'video':
+    case 'article':
+      return true;
+    default:
+      return false;
+  }
```

Comment:
I'm not sure about args order with lodash fp, but:

```
const isExternalContent = type => includes(type, ['scorm', 'video', 'article']);
```

Reply:
done